Android/FirebaseJobDispatcher/source/Firebase.JobDispatcher/packages.config

```diff
@@ -8,9 +8,6 @@
   <package id="Xamarin.Android.Support.Media.Compat" version="25.3.1" targetFramework="monoandroid71" />
   <package id="Xamarin.Android.Support.v4" version="25.3.1" targetFramework="monoandroid71" />
   <package id="Xamarin.Build.Download" version="0.4.5" targetFramework="monoandroid71" />
-  <package id="Xamarin.GooglePlayServices.Base" version="42.1021.1" targetFramework="monoandroid71" />
   <package id="Xamarin.GooglePlayServices.Basement" version="42.1021.1" targetFramework="monoandroid71" />
-  <package id="Xamarin.GooglePlayServices.Gcm" version="42.1021.1" targetFramework="monoandroid71" />
-  <package id="Xamarin.GooglePlayServices.Iid" version="42.1021.1" targetFramework="monoandroid71" />
   <package id="Xamarin.GooglePlayServices.Tasks" version="42.1021.1" targetFramework="monoandroid71" />
```

Comment:
I think we may have covered this before.  Apologies if that is the case.  

Do we need Xamarin.GooglePlayServices.Tasks and Xamarin.GooglePlayServices.Basement here?  If so then they should be added to the nuspec as a dependency and the sample nuget list to be complete??